src/demo/examples/visualization-bar/index.js

```diff
@@ -0,0 +1,90 @@
+
+import { visualizationBar, VisualizationBar, VisualizationBarSizes } from 'hexagon-js';
+import { div } from 'utils/selection';
+
+const randNum = max => Math.round(Math.random() * max);
+const rand = max => e => {
+  e.progress(randNum(max));
+};
+
+export default () => {
+  const selectionWithMax = div();
+  const selectionErrored = div();
+  const selectionWithoutMax = div();
+  const selectionWithContent = div();
+  const selectionWithLabels = div();
+  const selectionSmall = div();
```

Comment:
I'd just use the `div()` inline instead of creating these `const` values to keep the script tidy